tests/prague/eip6206_jumpf/test_jumpf_execution.py

```diff
@@ -0,0 +1,71 @@
+"""
+Execution of CALLF, RETF opcodes within EOF V1 containers tests
+"""
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Environment,
+    StateTestFiller,
+    TestAddress,
+    Transaction,
+)
+from ethereum_test_tools.eof.v1 import Container
+
+from .contracts import VALID, container_name
+from .spec import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-6206.md"
+REFERENCE_SPEC_VERSION = "a1775816657df4093787fb9fe83c2f7cc17ecf47"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "container",
+    VALID,
```

Comment:
@marioevz  imho this is a bit confusing when we import vectors from somewhere external file, where the actual test logic is hidden even more in a copuple of functions

Reply:
This no longer occurs in this PR.  I merged the eof and state tests in one pass.